src/main/java/ua/shpp/eqbot/command/AddService.java

```diff
@@ -15,13 +14,14 @@
 import ua.shpp.eqbot.service.SendBotMessageService;
 
 import static ua.shpp.eqbot.model.PositionMenu.MENU_START;
+
```

Comment:
why component 
what is the difference between component and service
i mean here service need

Reply:
The code currently in the developer does not use...